recipes/so5extra/all/conanfile.py

```diff
@@ -20,7 +20,9 @@ def _source_subfolder(self):
         return "source_subfolder"
 
     def requirements(self):
-        if tools.Version(self.version) >= "1.5.0":
+        if tools.Version(self.version) >= "1.5.1":
+            self.requires("sobjectizer/5.7.4")
+        elif tools.Version(self.version) >= "1.5.0":
             self.requires("sobjectizer/5.7.3")
         else:
             self.requires("sobjectizer/5.7.2.3")
```

Comment:
Can't it be `sobjectizer/5.7.4` for all `soextra` versions?

Reply:
done